homeassistant/components/halohome/config_flow.py

```diff
@@ -0,0 +1,57 @@
+"""Create a config flow for the HALO Home integration."""
+import logging
+from typing import List, Optional
+
+import halohome
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+import homeassistant.helpers.config_validation as cv
+
+from .const import CONF_LOCATIONS, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Required(CONF_PASSWORD): cv.string,
+        vol.Optional(CONF_HOST, default="https://api.avi-on.com"): cv.string,
```

Comment:
Would the user need to set a different host here?

Reply:
Potentially yes. HALO Home uses avi-on under the hood, but also a separate url that they sometimes use.

Reply:
how would an end-user know what URL to use?

Reply:
That's a good question that I don't really have an answer to. HALO Home is not very transparent so it's not clear in which situations a different URL is needed. I'm fine removing that as an option, since it might never come up.

Reply:
In that case, I would leave it out and see what happens in the future. It can always be added when a clear use-case pops up.